### PR TITLE
Selector for new review comments

### PR DESCRIPTION
**Pull request by OstrichProjects**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "Tab Size on GitHub",
-	"version": "1.1.2",
+	"version": "1.1.3",
```
Comment (sindresorhus) [inline manifest.json]: Don't bump version in a PR. It's up to the maintainer to decide when to release, what version, and how. I personally use automated script for this.

**Comment by sindresorhus** [comment]
What exactly does this apply to?

File: manifest.json (hunk 1 above)
Comment (OstrichProjects) [inline manifest.json]: These blocks of code are what this PR applies to.  They're new with the new review/single comment feature introduced by github.

**Comment by OstrichProjects** [comment]
If you look at the code in the outdated diff with the extension, the tabs are 8 spaces.  This PR fixes that.

**Comment by sindresorhus** [comment]
Thanks for clarifying :)

**Pull request merged**